examples/custom-resources/externalname-services/virtual-server/external-svc-dep.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: v1
```

Comment:
Can we either remove this folder for the virtual server external name service example, or complete the example with a readme and the required yaml files?

Reply:
@jjngx  I'd say we can  remove it for now, can do a separate PR for this later on.

Reply:
done, removed the VS folder